diffengine/__init__.py

```diff
@@ -149,14 +149,15 @@ def get_latest(self):
         """
 
         # make sure we don't go too fast
+        # TODO: can we remove this? Why is this here?
```

Comment:
@edsu the `time.sleep` here has a particular purpose? I've tried the script without it and the process is quite faster. Plus, I didn't face any troubles without it. Maybe we can erase it from the code...

Reply:
I put it there to prevent getting blocked by websites that saw repeated and rapid crawling as a threat. But honestly I had kind of forgotten about it. It might be nice to retain it as a configuration option, and have it default to zero?

Reply:
Yes, I like that. Will create that config in a separate PR!